examples/wasm-emscripten/src/main.rs

```diff
@@ -0,0 +1,111 @@
+#![no_main]
+
+// Embed models into the .wasm file.
+#[derive(rust_embed::RustEmbed)]
+#[folder = "models/"]
+pub struct Models;
+
+// Below YoloV8 implementation partly copied from the "yolov8" example.
+#[derive(Debug, Clone, Copy)]
+struct BoundingBox {
+	x1: f32,
+	y1: f32,
+	x2: f32,
+	y2: f32
+}
+
+#[rustfmt::skip]
+static YOLOV8_CLASS_LABELS: [&str; 80] = [
+	"person", "bicycle", "car", "motorcycle", "airplane", "bus", "train", "truck", "boat", "traffic light",
+	"fire hydrant", "stop sign", "parking meter", "bench", "bird", "cat", "dog", "horse", "sheep", "cow", "elephant",
+	"bear", "zebra", "giraffe", "backpack", "umbrella", "handbag", "tie", "suitcase", "frisbee", "skis", "snowboard",
+	"sports ball", "kite", "baseball bat", "baseball glove", "skateboard", "surfboard", "tennis racket", "bottle",
+	"wine glass", "cup", "fork", "knife", "spoon", "bowl", "banana", "apple", "sandwich", "orange", "broccoli",
+	"carrot", "hot dog", "pizza", "donut", "cake", "chair", "couch", "potted plant", "bed", "dining table", "toilet",
+	"tv", "laptop", "mouse", "remote", "keyboard", "cell phone", "microwave", "oven", "toaster", "sink", "refrigerator",
+	"book", "clock", "vase", "scissors", "teddy bear", "hair drier", "toothbrush"
+];
+
+#[no_mangle]
+pub extern "C" fn alloc(capacity: usize) -> *mut std::os::raw::c_void {
+	let mut buf = Vec::with_capacity(capacity);
```

Comment:
Allocating through `Vec` here is a little confusing, what about using [`std::alloc::alloc`](https://doc.rust-lang.org/stable/std/alloc/fn.alloc.html) directly?